### PR TITLE
update centos7 install for php 7.3

### LINKED ISSUE #92: Wrong PHP install guide for CentOS
On the page https://pterodactyl.io/community/installation-guides/panel/centos7.html#php-7-2 it says to install php with the follwoing command:
`yum install -y php72u-php php72u-common php72u-fpm php72u-cli php72u-json php72u-mysqlnd php72u-mcrypt php72u-gd php72u-mbstring php72u-pdo php72u-zip php72u-bcmath php72u-dom php72u-opcache`

However there is no `php72u-mcrypt` package for php v7.2. Mcrypt has been removed from php 7.2.
The `php72u-php` package is also not available.
See https://dl.iuscommunity.org/pub/ius/stable/CentOS/7/x86_64/repoview/letter_p.group.html

Please fix the documentation if possible. How do I replace these packages?

### PR DESCRIPTION
**Pull request by parkervcp**
move to using the remi repo
install php 7.3

This would resolve #92

### CODE REVIEW

**Comment by parkervcp** [comment]
I apparently left in some selinux stuff so this would supersede  #104 as well.

File: community/installation-guides/panel/centos7.md [hunk 1]
```diff
@@ -41,12 +41,15 @@ We recommend the ius repo to get the latest php packages.
...
+yum install -y yum-utils
+yum-config-manager --disable remi-php54
+yum-config-manager --enable remi-php73
 
 ## Get yum updates
 yum update -y
 
 ## Install PHP 7.3
-yum install -y php73-php php73-php-common php73-php-fpm php73-php-cli php73-php-json php73-php-mysqlnd php73-php-mcrypt php73-php-gd php73-php-mbstring php73-php-pdo php73-php-zip php73-php-bcmath php73-php-dom php73-php-opcache
+yum install -y php-php php-common php-fpm php-cli php-json php-mysqlnd php-mcrypt php-gd php-mbstring php-pdo php-zip php-bcmath php-dom php-opcache
```
Comment (Nuc1eoN) [inline community/installation-guides/panel/centos7.md]: `php-php` does not exist. Mistake?

Comment (Nuc1eoN) [reply community/installation-guides/panel/centos7.md]: Also `php-mcrypt` has been replaced by `php-pecl-mcrypt` btw

**Comment by Nuc1eoN** [comment]
Thanks for the work:)
I have just noticed that I did not even have `php-opcache` and `php-mcrypt` (actually `php-pecl-mcrypt`) installed, but my panel worked fine. Are they mandatory?

I did however install `php-pecl-zip` becasue it was required by comsposer, is it interchangable with `php-pecl-mcrypt`?

File: community/installation-guides/panel/centos7.md (hunk 1 above)
Comment (parkervcp) [reply community/installation-guides/panel/centos7.md]: missed that I had php-php in there.  This worked to install on my test VM I set up still. I assume it's due to php being a require package for the others and remi probably maps php-mcrypt to php-pecl-mcrypt

**Pull request closed without merging**